### PR TITLE
andreyt4265

### PR DESCRIPTION
**Pull request by andreyt4265**

### CODE REVIEW

**Comment by chfast** [comment]
What is this about?

File: libethereum/BlockQueue.cpp [hunk 1]
```diff
@@ -52,9 +52,10 @@ BlockQueue::BlockQueue():
 {
 	// Allow some room for other activity
 	unsigned verifierThreads = std::max(thread::hardware_concurrency(), 3U) - 2U;
+	verifierThreads = 1;
```
Comment (pirapira) [inline libethereum/BlockQueue.cpp]: Now the expression `std::max(...) - 2U` is not used anymore.

Maybe you can replace the initialization.  Maybe you can remove the variable `verifierThreads` altogether.

**Comment by pirapira** [comment]
@andreyt4265 change the title of the PR to something descriptive, and fill in the first comment (now empty) in this PR with the purpose of the change.

**Comment by gumb0** [comment]
This looks abandoned.

**Pull request closed without merging**